source/slang/slang-ir-inline.h

```diff
@@ -11,7 +11,7 @@ namespace Slang
     class DiagnosticSink;
 
         /// Any call to a function that takes or returns a string parameter is inlined
-    Result performStringInlining(IRModule* module, DiagnosticSink* sink);
+    Result performTypeInlining(IRModule* module, DiagnosticSink* sink);
```

Comment:
Maybe you can update this comment to reflect that it works also for ref type now?